development/webpack/utils/loaders/swcLoader.ts

```diff
@@ -201,6 +223,11 @@ export function getSwcLoader(
         parser: {
           syntax,
           [syntax === 'typescript' ? 'tsx' : 'jsx']: enableJsx,
+          importAssertions: true,
+        },
+        experimental: {
+          keepImportAttributes: true,
+          emitAssertForImportAttributes: true,
```

Comment:
unsure if all three or only some are needed, but definitely something needs to be set to deal with the import assertion

Reply:
`import config from "./external-extension-config.json" assert { type: "json" };`